eng/scripts/Publish-MavenPackages.ps1

```diff
@@ -264,8 +264,8 @@ foreach ($packageDetail in $packageDetails) {
   $stagingDescriptionOption = "-DstagingDescription=$($packageDetail.FullyQualifiedName)"
   Write-Information "Staging Description Option is: $stagingDescriptionOption"
 
-  if ($RepositoryUrl -like "https://pkgs.dev.azure.com/azure-sdk/public/*") {
-    Write-Information "GPG Signing and deploying package in one step to: $RepositoryUrl"
+  if ($RepositoryUrl -match "https://pkgs.dev.azure.com/azure-sdk/\b(internal|public)\b/*") {
+    Write-Information "GPG Signing and deploying package in one step to public feed: $RepositoryUrl"
```

Comment:
Should this be "public feed" -> "devops feed"?

Reply:
Since it's no longer just publishing to the public feed, yes. I'll get it updated.